server/jetstream_api.go

```diff
@@ -1024,6 +1026,12 @@ func (s *Server) jsStreamListRequest(sub *subscription, c *client, subject, repl
 		offset = req.Offset
 	}
 
+	// Clustered mode will invoke a scatter and gather.
+	if s.JetStreamIsClustered() {
+		s.startGoRoutine(func() { s.jsClusteredStreamListRequest(acc, ci, offset, subject, reply, msg) })
```

Comment:
Don't we need to copy some of the content of those variables here? Meaning when the stack unrolls, those fields may be overwritten (say msg points to wire, etc..).

Reply:
Ah I see what you mean. Will take a look more closely. Thanks.